### PR TITLE
Test against PostgreSQL 13

### PR DESCRIPTION
**Pull request by pimeys**
Closes https://github.com/prisma/prisma-engines/issues/755

### CODE REVIEW

File: libs/test-setup/src/connectors.rs [hunk 1]
```diff
@@ -15,6 +15,7 @@ fn connector_names() -> Vec<(&'static str, Tags)> {
         ("postgres", Tags::POSTGRES),
         ("postgres11", Tags::POSTGRES),
         ("postgres12", Tags::POSTGRES),
+        ("postgres13", Tags::POSTGRES),
```
Comment (tomhoule) [inline libs/test-setup/src/connectors.rs:18]: I'm kind of surprised it's enough to define a new connector for the whole test macros infrastructure, I didn't remember I made it that easy :p

Comment (pimeys) [reply libs/test-setup/src/connectors.rs:18]: There are a few test api functions also that were a bit of a drag to add...

Comment (tomhoule) [reply libs/test-setup/src/connectors.rs:18]: Yeah, but I think these can't be avoided if we want control.

Comment (tomhoule) [reply libs/test-setup/src/connectors.rs:18]: Maybe we could reduce the boilerplate for these with some `macro_rules!` magic, or even a full-on proc macro.

Comment (tomhoule) [reply libs/test-setup/src/connectors.rs:18]: If we start testing more versions of databases, for example.

**Pull request merged**